tests/core/actions/test_forms.py

```diff
@@ -90,10 +92,14 @@ async def test_activate_with_prefilled_slot():
         mappings:
         - type: from_entity
           entity: {slot_name}
+          conditions:
+          - active_loop: {form_name}
       {next_slot_to_request}:
         type: text
         mappings:
         - type: from_text
+          conditions:
+          - active_loop: {form_name}
```

Comment:
should the test be duplicated instead?

Reply:
Yep, good point 👍🏼

Reply:
done: https://github.com/RasaHQ/rasa/pull/11149/commits/8c87ff1eac90aca80a73d27c3d56afbb8e9bae74